lib/chef/knife/core/bootstrap_context.rb

```diff
@@ -190,13 +190,24 @@ def knife_config
         # chef version string to fetch the latest current version from omnitruck
         # If user is on X.Y.Z, bootstrap will use the latest X release
         def latest_current_chef_version_string
-          chef_version_string = if knife_config[:bootstrap_version]
-                                  knife_config[:bootstrap_version]
-                                else
-                                  Chef::VERSION.split(".").first
-                                end
+          installer_version_string = nil
+          if @config[:prerelease]
+            installer_version_string = ["-p"]
+          else
+            chef_version_string = if knife_config[:bootstrap_version]
+                                    knife_config[:bootstrap_version]
+                                  else
+                                    Chef::VERSION.split(".").first
+                                  end
+
+            installer_version_string = ["-v", chef_version_string]
 
-          "-v #{chef_version_string}"
+            # If bootstrapping a pre-release version add -p to the installer string
+            if chef_version_string.split(".").length > 3
+              installer_version_string << "-p"
+            end
```

Comment:
We no longer make "prerelease" gems (x.y.z.pre.0, x.y.z.beta.0) and just use channels now, so this conditional can be dropped.